connect_ext_test.go

```diff
@@ -1393,6 +1467,305 @@ func TestBidiStreamServerSendsFirstMessage(t *testing.T) {
 	})
 }
 
+func TestStreamForServer(t *testing.T) {
+	t.Parallel()
+	newPingServer := func(pingServer pingv1connect.PingServiceHandler) (pingv1connect.PingServiceClient, *httptest.Server) {
+		mux := http.NewServeMux()
+		mux.Handle(pingv1connect.NewPingServiceHandler(pingServer))
+		server := httptest.NewUnstartedServer(mux)
+		server.EnableHTTP2 = true
+		server.StartTLS()
+		t.Cleanup(server.Close)
```

Comment:
The subtests all call `t.Cleanup(server.Close)` with their sub-t's, so we don't need this here.